docs/15-tamagotchi-shop/tamagotchi-shop.md

```diff
@@ -150,16 +153,16 @@ extern "C" fn init() {
 }
 ```
 
-Note, that here we use `async fn main()` with `#[gstd::async_main]` macros instead of `handle()` function. When asynchronous functions appear in the contract, the async main function becomes the program entry point.
+**NOTE:** We use the `async fn main()` syntax along with the `#[gstd::async_main]` macros instead of the `handle()` function. When we have asynchronous functions in our contract, the async main function becomes the starting point of our program.
```

Comment:
```suggestion
:::note

We use the `async fn main()` syntax along with the `#[gstd::async_main]` macros instead of the `handle()` function. When we have asynchronous functions in our contract, the async main function becomes the starting point of our program.

:::
```